docs/remix_as_code_viewer.md

```diff
@@ -0,0 +1,38 @@
+Remix as code viewer
+=============================
+
+Through Etherscan
+------------------
+
+If there is a contract verified on Etherscan and someone wants to have a look on it in an IDE, mostly for a mutiple part contract verification, Remix provides a quick way to load the contract.
```

Comment:
I got a bit lost in your first sentence.  What do you think of this edit:
Verified contracts on Etherscan can be viewed in Remix by making a simple change to the URL.

Reply:
ahh but here I'm using "simple"...(see below)

Reply:
updated